src/alto-ui/Actions/__tests__/Actions.test.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import Actions from '../Actions';
+import Dropdown from '../../Dropdown';
+import Group from '../../Group';
+
+jest.mock('../../Dropdown', () => {
+  return props => {
+    return (
+      <div role="menu" tabIndex="0">
+        {props.items.map(action => {
+          return (
+            <button key={`key_${action.key}`} id={`key_${action.key}`} onClick={action.onClick || props.onClick} />
+          )
+        })}
+      </div>
+    )
+  };
+});
+
+jest.mock('../../Group', () => {
+  return props => {
+    return (
+      <div role="menu" tabIndex="0" items={props.items} />
+    )
+  };
+});
+
+describe('Actions', () => {
```

Comment:
I want a test that `Actions` returns `null` if no items.

Reply:
Added `it('returns null if no items provided', () =>` test